.github/workflows/nightly_docs.yml

```diff
@@ -40,14 +40,14 @@ jobs:
           repository: bitcoindevkit/bitcoindevkit.org
           ref: master
       - name: Create directories
-        run: mkdir -p ./docs/.vuepress/public/docs-rs/bdk/nightly
+        run: mkdir -p ./docs/.vuepress/public/docs-rs/wallet/nightly
       - name: Remove old latest
-        run: rm -rf ./docs/.vuepress/public/docs-rs/bdk/nightly/latest
+        run: rm -rf ./docs/.vuepress/public/docs-rs/wallet/nightly/latest
       - name: Download built docs
         uses: actions/download-artifact@v1
         with:
           name: built-docs
-          path: ./docs/.vuepress/public/docs-rs/bdk/nightly/latest
+          path: ./docs/.vuepress/public/docs-rs/wallet/nightly/latest
```

Comment:
Somehow this seems counter-intuitive.
The `crate` is `bdk` that contains sub-crates `wallet`, `chain` etc.

Can we keep the crate as `bdk` with the `bdk_wallet`, `bdk_chain` etc sub-crates?

Maybe in a future we might use a prelude, i.e. `use bdk::prelude::*`.
Then we are shooting ourselves in the foot again, because it would be a breaking change.

Reply:
I think we need to add links to the web page with reach of the crates that are part of the `bdk` repo. I'd rather do this web docs link fix in a separate PR for the beta milestone. 📌